testing/src/test/java/org/motechproject/nms/testing/it/kilkari/InboxServiceBundleIT.java

```diff
@@ -271,4 +273,85 @@ public void verifyFT122() throws Exception {
 
 	}
 	
+	/*
+	 *	"To check NMS is able to make available a single message of current week in inbox
+	 *	 when user is subscribed to 72Weeks Pack with single message per week configuration."
+	 */
+	@Test
+	public void verifyFT107() throws NoInboxForSubscriptionException {
+		
+		DateTime now = DateTime.now();
+		// create subscriber for pregnancyPack
+		Subscriber subscriber = new Subscriber(1000000002L);
+		subscriber.setLastMenstrualPeriod(now.minusDays(90));
+		subscriberService.create(subscriber);
+		
+		// create pregnancyPack subscription for one message per week.
+		subscriptionService.createSubscription(subscriber.getCallingNumber(), rh.hindiLanguage(), 
+				sh.pregnancyPackFor1MessagePerWeek(subscriptionPackMessageDataService), SubscriptionOrigin.MCTS_IMPORT);
+		
+		subscriber = subscriberService.getSubscriber(subscriber.getCallingNumber());
+		Set<Subscription> subscriptions = subscriber.getAllSubscriptions();
+		Subscription subscription = subscriptions.iterator().next();
+		SubscriptionPackMessage msg = inboxService.getInboxMessage(subscription);
+		
+		// first msg should be in inbox
+		assertEquals(msg.getWeekId(), "w1_1");
+		assertEquals(msg.getMessageFileName(), "w1_1.wav");
+		// set lmp as 7th day week of week to check message exist for seven days.
+		subscriber.setLastMenstrualPeriod(DateTime.now().minusDays(96)); 
+		subscriberService.update(subscriber);
+
+		subscriber = subscriberService.getSubscriber(subscriber.getCallingNumber());
+		subscriptions = subscriber.getAllSubscriptions();
+		subscription = subscriptions.iterator().next();
+		msg = inboxService.getInboxMessage(subscription);
+
+		// still first message should be in inbox because messagePerWeek is one
+		assertEquals(msg.getWeekId(), "w1_1");
+		assertEquals(msg.getMessageFileName(), "w1_1.wav");
+		
+	}
+	
+	/*
+	 *	"To check NMS is able to make available a single message of current week in inbox
+	 *	 when user is subscribed to 48Weeks Pack with single message per week configuration."
+	 */
+	@Test
+	public void verifyFT114() throws NoInboxForSubscriptionException {
+		
+		DateTime now = DateTime.now();
+		// create subscriber for childPack
+		Subscriber subscriber = new Subscriber(1000000002L);
+		subscriber.setDateOfBirth(now);
+		subscriberService.create(subscriber);
+		
+		// create childPack subscription for two message per week.
```

Comment:
The description above says single message per week, yet here you set two messages per week.
What is it supposed to really be?